README.md

```
@@ -1,142 +1,125 @@
 
-- Strict security headers (`COOP`/`COEP`, CSP, `nosniff`, `X-Frame-Options`,
-  `microphone=()`) via `server/securityHeaders.js` in development and
-  `vercel.json` in production
+- Strict headers (`COOP`/`COEP`, CSP, `nosniff`, `X-Frame-Options`, `microphone=()`) via `server/securityHeaders.js` and `vercel.json`
 - All secrets via environment variables — see `.env.example`
 - ONNX models verified against pinned SHA-256 hashes before every session
 
 ## License
 
-UNLICENSED — © Randy Jordan. All rights reserved.
+UNLICENSED — © Randy Jordan. All rights reserved.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)